style.css

```diff
@@ -234,12 +243,116 @@
     align-items: center;
   }
 
-  .grid-3-movile {
+  .projectCards {
     position: relative;
     width: 100%;
     top: 139px;
   }
 
+  .contPopUp {
+    z-index: 10;
+    display: none;
+    position: fixed;
+    padding: 10%;
+    color: black;
+    width: 100vw;
+    height: 100vh;
+    box-sizing: border-box;
+    backdrop-filter: blur(5px);
+  }
+
+  .popUp-main {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  .contentPopUp {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    width: 100%;
+    height: 85vh;
+    padding: 0% 5% 5% 5%;
+    box-sizing: border-box;
+  }
```

Comment:
- [ ] You have done a great job  on this milestone, and according to the previous reviewer ,please you have not implement the changes  requested by the previouse reviewer.the mobile version currently has an overflow which is as a result of using excess height for the popup hence it doesn't display the full mobile popup. please effect this change so as to have a better interface and user friendly 😄
- [ ] your desgn
![Screenshot 2023-05-19 223238](https://github.com/Zven94/portfolio/assets/92458236/e6c6843b-55eb-4274-907e-2c8271a40f37)

- [ ] Figma design
![Screenshot 2023-05-19 223613](https://github.com/Zven94/portfolio/assets/92458236/131921c0-c303-40d7-8628-d8e01d548384)